classes/migration/upgrade/v3_5_0/I8333_AddMissingForeignKeys.php

```diff
@@ -0,0 +1,253 @@
+<?php
+
+/**
+ * @file classes/migration/upgrade/v3_5_0/I8333_AddMissingForeignKeys.php
+ *
+ * Copyright (c) 2023 Simon Fraser University
```

Comment:
`2024`

Reply:
This code has aged, but I'll update haha.